atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -157,39 +163,50 @@ public long getSweepPersistentLockWaitMillis() {
 
     /**
      * The target number of (cell, timestamp) pairs to examine in a single run of the background sweeper.
+     * @deprecated Use {@link AtlasDbRuntimeConfig#getSweepReadLimit} to make this value
+     * live-reloadable.
      */
     // TODO(gbonik): make this Default after we delete the deprecated options. For now, we need to be able to detect
     // whether the field is present in the configuration file.
+    @Deprecated
     @Nullable
     public abstract Integer getSweepReadLimit();
 
     /**
      * The target number of candidate (cell, timestamp) pairs to load per batch while sweeping.
+     * @deprecated Use {@link AtlasDbRuntimeConfig#getSweepCandidateBatchHint} to make this value
+     * live-reloadable.
      */
     // TODO(gbonik): make this Default after we delete the deprecated options. For now, we need to be able to detect
     // whether the field is present in the configuration file.
+    @Deprecated
     @Nullable
     public abstract Integer getSweepCandidateBatchHint();
 
     /**
      * The target number of (cell, timestamp) pairs to delete at once while sweeping.
+     * @deprecated Use {@link AtlasDbRuntimeConfig#getSweepDeleteBatchHint} to make this value
+     * live-reloadable.
      */
     // TODO(gbonik): make this Default after we delete the deprecated options. For now, we need to be able to detect
     // whether the field is present in the configuration file.
+    @Deprecated
     @Nullable
     public abstract Integer getSweepDeleteBatchHint();
 
     /**
-     * @deprecated Use {@link #getSweepReadLimit()}, {@link #getSweepCandidateBatchHint()}
-     * and {@link #getSweepDeleteBatchHint()} instead.
+     * @deprecated Use {@link AtlasDbRuntimeConfig#getSweepReadLimit()},
+     * {@link AtlasDbRuntimeConfig#getSweepCandidateBatchHint()} and
+     * {@link AtlasDbRuntimeConfig#getSweepDeleteBatchHint()}.
      */
     @Deprecated
```

Comment:
sorry, i didn't understand why we can't get rid of these deprecated fields and make the others default. this is a new object - so we can just force people to use the new configs when they migrate to it.